.finalizeRelease.sh

```diff
@@ -1,35 +1,84 @@
 #!/bin/bash
 
 set -o errexit # Exit on error
-readonly UI_VERSION="v0.4.8-beta"
+readonly VERSION="v0.4.8-beta"
+declare -a Electron_Archs=("linux-x64" "linux-ia32" "win32-x64" "win32-ia32" "darwin-x64")
+declare -a Sia_Archs=("linux-amd64" "linux-386" "windows-amd64" "windows-386" "darwin-amd64")
 
-cd "release/$UI_VERSION"
+cd "release/$VERSION"
 
-# Array of different distribution strings
-declare -a OSes=("linux-x64" "linux-ia32" "win32-x64" "win32-ia32" "darwin-x64")
+# generic packaging function
+package() {
+	local electron_arch="$1"
+	local ui="Sia-UI-$electron_arch"
+	local extension=".zip"
+	if [ "${1:0:5}" == "linux" ]; then
+		extension=".tar.gz"
+	fi
```

Comment:
It makes it so I can do the following series:
```
local sia_arch=""
for i in "${!Electron_Archs[@]}"; do
   if [[ "${Electron_Archs[$i]}" = "${1}" ]]; then
       sia_arch="${Sia_Archs[$i]}";
   fi
done
local sia_folder="Sia-${VERSION}-${sia_arch}"
local sia_archive="${sia_folder}${extension}"
local sia_url="https://github.com/NebulousLabs/Sia/releases/download/${VERSION}/${sia_archive}"
```
to get the one-line of wget (instead of 3):
```bash
wget -q $sia_url
```